### DISCUSSION TITLE
Thank you for the first CSL2 mod for traffic light improvement. Look forward for next improvement / TMPE

### DISCUSSION THREAD

**Discussion by jackfood** [slyh/Cities2-TrafficLightsEnhancement#2] [General]

**Comment by slyh** [comment]
Thank you for your support.

The current scope of the project is to enable players to select from
several pre-defined signal patterns, for which I have a prototype
running. Additionally, I am considering a rudimentary lane
modification system.

I have enabled the Discussions section, so if you have any
suggestions, please feel free to post them there.

**Reply by jackfood** [reply]
I'm glad to hear that this feature is something the community truly needs. It's fantastic that the ability to edit traffic light phasing (timing) and define lane choices hope to be in the works. We can't thanks you enough and also realized it is already released as v1.2.

**Comment by Zaltocleotl** [comment]
If I could pay you what the work is worth I would. I could afford a cup of coffee or such. This kind of modding is awesome! And you're doing it quickly and cleanly. It's not perfectly stable, but it's an alpha I've followed since first release, and it works and needs little interjection to work. I love the newest UI and lane controls! 
Thank you!
